internal/pkg/agent/install/uninstall.go

```diff
@@ -116,6 +116,14 @@ func Uninstall(cfgFile, topPath, uninstallToken string, log *logp.Logger, pt *pr
 		pt.Describe("Successfully uninstalled service")
 	}
 
+	// Removing the external logger isn't fatal to uninstall
+	err = removeExternalLogger()
```

Comment:
Should this really be done? Is it harmful to keep it around? Upon performing uninstall does it remove the entries, or does this still keep the entries?

With https://github.com/elastic/elastic-agent/pull/4822 two Elastic Agent's will be installed at once. In this case if only one is uninstalled then this will be removed and the still installed Elastic Agent will not be able to log.

Reply:
This would definitely break in the 2 agent case.  The Remove just removes the registry entries, so not a big deal to leave them around.  I'll remove.